cps/page.py

```diff
@@ -0,0 +1,39 @@
+import os
+import flask
+import markdown
+from flask import abort
+from pathlib import Path
+from flask_babel import gettext as _
+
+from . import logger, config, ub
+from .render_template import render_title_template
+from .constants import CONFIG_DIR as _CONFIG_DIR
+
+page = flask.Blueprint('page', __name__)
+
+log = logger.create()
+
+@page.route('/page/<string:file>', methods=['GET'])
+def get_page(file):
+    page = ub.session.query(ub.Page)\
+        .filter(ub.Page.name == file)\
+        .filter(ub.Page.is_enabled)\
+        .first()
+
+    if page:
+        dir_config_path = os.path.join(_CONFIG_DIR, 'pages')
+        file_name = Path(file + '.md')
+        file_path = dir_config_path / file_name
+
+        if file_path.is_file():
```

Comment:
NIT: you could use a try here instead:
and then after line 30 you could do 
```
    except FileNotFoundError:
        log.error(f"'{file}' was accessed but file doesn't exist.")
        abort(404)
 ```
 it just makes the code more readable